sdk/keyvault/azure-security-keyvault-keys/src/private/key_serializers.hpp

```diff
@@ -14,7 +14,7 @@
 #include "azure/keyvault/keys/deleted_key.hpp"
 #include "azure/keyvault/keys/import_key_options.hpp"
 #include "azure/keyvault/keys/key_vault_key.hpp"
-#include "azure/keyvault/keys/list_keys_result.hpp"
+#include "azure/keyvault/keys/list_keys_responses.hpp"
```

Comment:
The fact that this header also contains option **input** types that are used as input, and not just `PagedResponse<T>` **output**/return types, is a bit strange.

But, this change is better than what's in main atm :p